WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -53,7 +53,11 @@ final class CardPresentPaymentsOnboardingUseCase: CardPresentPaymentsOnboardingU
     }
 
     func refresh() {
-        if state != .completed {
+        var completed = false
+        if case .completed(_) = state { // Swift is weird
```

Comment:
It is 😁 When I experimented with [something similar](https://github.com/woocommerce/woocommerce-ios/pull/6212/files#diff-08df722e9250a12bcc6f7d83d64bbf3bb6da0e531a7c16088baac45b10bf996bR103), I added an `isComplete` computed property

Reply:
Capital idea. Fixed in 63977f16212b325aababf0d8822e9028b533a4ef